### ISSUE TITLE
some problems about finetune

### ISSUE THREAD

**Issue by tanggang1997** [Plachtaa/VITS-fast-fine-tuning#387]
When fine-tuning, I found that the loss function was reduced after a few epochs, but it was oscillating, and when I continued to train, I found that the synthesis was improved, so what is a situation where you train to the final loss?

**Comment by anfogy** [comment]
There's no a define of "final loss". But most finetune processes will soft lock the loss into a value, such as the loss oscillate for 20000 steps and drop to 5, then it locks on 5 or slightly drops to 4.997 or something like that

**Comment by tanggang1997** [comment]
> 
Thank you for your answer, another question is whether the tts pronunciation of your Chinese corpus is strange, the one I trained is very strange, and I found that noise_scale and noise_scale_w have some effect on the speaker's pronunciation, may I ask how these two parameters are controlled?

**Comment by anfogy** [comment]
> Thank you for your answer, another question is whether the tts pronunciation of your Chinese corpus is strange, the one I trained is very strange, and I found that noise_scale and noise_scale_w have some effect on the speaker's pronunciation, may I ask how these two parameters are controlled?

According to the definitions in `cmd_inference.py`, `noise_scale` is the emotion scale of the speaker and the `noise_scale_w` means how long the speaker pronounce each phoneme.
About how they are controlled, you can try to increase them by 0.1(about default of the value you can find in `cmd_inference.py`.

**Comment by anfogy** [comment]
If you think the pronunciation of Chinese is strange, try adding some Chinese data into your training data or use auxiliary dataset in the train.

**Comment by tanggang1997** [comment]
> If you think the pronunciation of Chinese is strange, try adding some Chinese data into your training data or use auxiliary dataset in the t
